<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DICOMIGTLWindow</class>
 <widget class="QMainWindow" name="DICOMIGTLWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="gpScanner">
       <property name="title">
        <string>Scanner</string>
       </property>
       <widget class="QPushButton" name="pbScannerActivate">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>201</y>
          <width>261</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Activate</string>
        </property>
       </widget>
       <widget class="QWidget" name="formLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>261</width>
          <height>101</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="controlPortLabel">
           <property name="text">
            <string>Control Port #:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="leControlPort"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="scannerAddressLabel">
           <property name="text">
            <string>Scanner Addr:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="leScannerAddress"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="imagePortLabel">
           <property name="text">
            <string>Image Port #</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="leImagePort"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>261</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="leStatusControl">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leStatusImage">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gpOpenIGTLink">
       <property name="title">
        <string>OpenIGTLink</string>
       </property>
       <widget class="QPushButton" name="pbClientActivate">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>201</y>
          <width>261</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Activate</string>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>261</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="leStatusClient">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="formLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>261</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::FieldsStayAtSizeHint</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="portLabel">
           <property name="text">
            <string>Port #:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="leOpenIGTLinkPort"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pbQuit">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>250</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAbout">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>About</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
